packages/docs/docusaurus.config.js

```diff
@@ -8,7 +8,7 @@ const watch = ['y', 'yes', 't', 'true', '1'].includes(process.env.TYPEDOC_WATCH)
 module.exports = {
   title: 'Temporal Node.js SDK API Reference',
```

Comment:
May as well fix the title, I've been wondering where this title is coming from...